packages/core/src/fhir-to-cda/cda-templates/constants.ts

```diff
@@ -47,3 +47,6 @@ export const codeSystemAttribute = "@_codeSystem";
 export const codeSystemNameAttribute = "@_codeSystemName";
 export const displayNameAttribute = "@_displayName";
```

Comment:
I think we can do the same thing with all these other attributes here too. The default prefix is I think is @_ and if we make it _, then we can use standard indexing
See: https://github.com/NaturalIntelligence/fast-xml-parser/blob/master/docs/v4/2.XMLparseOptions.md
and https://github.com/metriport/metriport/blob/031d29a3ce1abcd330c9bfb8dca0902dc8c71f23/packages/core/src/external/carequality/ihe-gateway-v2/outbound/xca/process/dq-response.ts#L160